nuxeo-platform-video-core/src/main/java/org/nuxeo/ecm/platform/video/service/VideoServiceImpl.java

```diff
@@ -191,7 +191,9 @@ public VideoConversionStatus getProgressStatus(String repositoryName, String doc
         WorkManager workManager = Framework.getLocalService(WorkManager.class);
         Work work = new VideoConversionWork(repositoryName, docId, conversionName);
         State state = workManager.getWorkState(work.getId());
-        if (state == State.SCHEDULED) {
+        if (state == null) { // DONE
+            return null;
```

Comment:
This was the code before that change: https://github.com/nuxeo/nuxeo-platform-video/commit/c10a8918c6e9c3b5ad7a692b332d517cf0ca9208#diff-73415684fc414055ea92718b709ddd70L194

I don't want to refactor everything, just put back what was working.